code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -7265,17 +7264,32 @@ public PlayerCharacter clone()
 		}
 
 		aClone.levelInfoFacet.removeAll(aClone.id);
-		for (PCLevelInfo info : getLevelInfo())
+		try
 		{
-			PCLevelInfo newLvlInfo = info.clone();
-			aClone.levelInfoFacet.add(aClone.id, newLvlInfo);
+			for (PCLevelInfo info : getLevelInfo())
+			{
+				PCLevelInfo newLvlInfo = info.clone();
+				aClone.levelInfoFacet.add(aClone.id, newLvlInfo);
+			}
+		}
+		catch (CloneNotSupportedException e)
```

Comment:
This should not be subsumed.  Those who call clone should know the risks... besides it's called in two places... not  that hard to address it where it's used.